docs/manuals/gui/callbacks.md

```diff
@@ -157,12 +155,12 @@ def on_action(state, id, action):
     invoke_long_callback(state, heavy_function, [...heavy_function arguments...])
 ```
 
-Line 5 shows the invocation of `invoke_long_callback()^` which does all the hard work. All the parameters
-to *heavy_function()* should be sent to the call, encapsulated in an array.<br/>
-When the call is performed, *heavy_function()* is invoked, but the application keeps running anyway:
-the user function is running in a background thread. 
+Line 5 shows the invocation of `invoke_long_callback()^` which does all the hard work. All the
+parameters to *heavy_function()* should be sent to the call, encapsulated in an array.<br/>
+When the call is performed, *heavy_function()* is invoked, but the application keeps running
+anyway: the user function is running in a background thread. 
```

Comment:
I might replace the semicolon with 'as' or a period.

Reply:
I'll do that, thx!